UPGRADE-3.x.md

```diff
@@ -1,13 +1,10 @@
 UPGRADE 3.x
 ===========
 
-### Deprecated `Sonata\CoreBundle\Form\EventListener\ResizeFormlistener::preBind`
-
-Use `Sonata\CoreBundle\Form\EventListener\ResizeFormlistener::preSubmit` instead.
-
-### Deprecated `Sonata\CoreBundle\Form\EventListener\ResizeFormlistener::onBind`
```

Comment:
Do we want to remove this?

Reply:
It was removed by @Soullivaneuh in a1b8371 . I don't remember having a conflict on that file.